README.md

```diff
@@ -224,6 +224,14 @@ open-attestation document-store verify --document ./examples/wrapped-documents/e
 ✔  success   The document is valid
 ```
 
+## DID Direct Signing
+
+Sign on an OA document directly with a private key.
+
+```bash
+open-attestation sign ./examples/unsigned-documents -f ./examples/sample-key -p did:ethr:0x6813Eb9362372EEF6200f3b1dbC3f819671cBA69#controller --output-dir ./examples/signed-documents
```

Comment:
I dunno if you want to handle that because we are quite inconsistent, but I needed to prepent my private key with 0x otherwise I got the following error: `✖  error     invalid hexlify value (argument="value", value="<value>", code=INVALID_ARGUMENT, version=bytes/5.0.3)`
`

Reply:
Yep, I've updated the example, the problem is that I dont want to prepend 0x to every private key.... we cannot assume an eth private key for all.

Reply:
yeah true, then nothing to do =)